### DISCUSSION TITLE
Загрузка приложения | Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#15] [Announcements]
# Загрузка приложения | Angular с примерами кода

При запуске приложения первым выполняется код, который определен в файле main.ts

https://angdev.ru/guide/basic/load-app/

**Comment by nessy126** [comment]
Скажите, пожалуйста, вся документация подходит к какой версии Angular?

**Reply by bndby** [reply]
9 и выше по идее

**Reply by nessy126** [reply]
А не знаете, 9 и 14 сильно отличаются?

**Reply by nessy126** [reply]
Не загрузка приложения, а вообще
